assets/js/blocks/product-query/constants.ts

```diff
@@ -9,6 +9,7 @@ import type { InnerBlockTemplate } from '@wordpress/blocks';
  */
 import { QueryBlockAttributes } from './types';
 import { VARIATION_NAME as PRODUCT_TITLE_ID } from './variations/elements/product-title';
+import { VARIATION_NAME as ADD_TO_CART_BUTTON_VARIATION_NAME } from './variations/elements/add-to-cart-button';
```

Comment:
Just for consistency with the other variables, I'd name this `ADD_TO_CART_BUTTON_ID`.

Reply:
Thanks for pointing this out. IMO It's better to add `VARIATION_NAME` suffix to make name more readable. For example in [this file](https://github.com/woocommerce/woocommerce-blocks/blob/0f2903ce4c5465d84d2f5f145e23d505a08f48ae/assets/js/blocks/product-query/index.tsx) also, we are using the same name `PRODUCT_TITLE_ID` but value it contains is core name.
What do you think about changing the names to add suffix `CORE_NAME` & `VARIATION_NAME` instead?

Reply:
I don't have a strong opinion about this. If you strongly feel that the further variable name length is paid off in readability, go for it! 😊  As long as we are using the same names consistently.

Reply:
Great! Thanks. I will update the names 🤝 🙌🏻